pkg/emb6/patches/0002-Rename-colliding-files-and-functions.patch

```diff
@@ -66,7 +66,7 @@ index 2912049..682b7c2 100644
  #include "ctimer.h"
 -#include "random.h"
 +#include "emb6_random.h"
- 
+
```

Comment:
Changing patch files this way can potentially end badly ;-)

Reply:
oops, you're right, that was unintentional. There are other patches with such, which I left out.